docs/proposals/multi-platform.md

```diff
@@ -0,0 +1,477 @@
+# Kubernetes for multiple platforms
+
+**Author**: Lucas Käldström ([@luxas](https://github.com/luxas))
+
+**Status** (17th of June 2016): Most parts are already implemented; still there's room for improvement
+
+## Abstract
+
+Kubernetes is written in Go, and Go code is portable across platforms.
+Docker and rkt are also written in Go, and it's already possible to use docker on various platforms.
+When it's possible to run docker containers on a specific architecture, people also want to use Kubernetes to manage the containers.
+
+We obviously want Kubernetes to run on as many platforms as possible.
+This is a proposal that explains what already has been done and why, and what's proposed to yet be done.
+
+With this work, we want to achieve a true cross-platform container management system.
+
+### TL; DR;
+
+This is a pretty long proposal with a lot of information and details, but here are the main tasks that are yet to be done:
+
+ - Standardize all Makefiles to build for all platforms
+   - Document in the devel documentation how to do this, and make people actually care about doing it
+ - Help Docker to fully implement the Manifest List, and implement it for all Kubernetes infrastructure images.
+   - To help Docker directly might be a bit out of topic for a Kubernetes proposal, so the first step is to use/build a small utility that is able to build and push Manifest list images.
+ - Make it possible to run a CI (for distributed testing) on other platforms
```

Comment:
Totally agree that we need to make it easy for anyone to setup CI, and that it should work on any platform that uses kubernetes... ideally by using kubernetes itself.